app/components/cases/submissions/status-pill.js

```diff
@@ -0,0 +1,46 @@
+import { service } from '@ember/service';
+import Component from '@glimmer/component';
+import { task } from 'ember-concurrency';
+import { tracked } from '@glimmer/tracking';
+import CONSTANTS from 'frontend-kaleidos/config/constants';
+
+/**
+ * @argument status submission status concept
+ * @argument meeting optional, what meeting the submission is connected to
+ */
+export default class CasesSubmissionsStatusPillComponent extends Component {
+  @service intl;
+  @service store;
+
+  @tracked label;
+  @tracked skin = 'ongoing';
+
+  constructor() {
+    super(...arguments);
+    this.loadStatusText.perform();
+  }
+
+  loadStatusText = task(async () => {
+    // await promises
+    const status = await this.args.status;
+    const meeting = await this.args.meeting;
+    // show status already
+    this.label = status.label || status.altLabel;
```

Comment:
Probably due to faulty data, but it appears `status` can be `null`. I'll add a check here.

![image](https://github.com/user-attachments/assets/fc448bc9-14a2-4efc-87c9-bfcc41c679ac)